roles/openshift_master/tasks/main.yml

```diff
@@ -91,8 +91,15 @@
   template:
     dest: "{{ item.filename }}"
     src: htpasswd.j2
-    mode: 0600
     backup: yes
+  when: item.kind == 'HTPasswdPasswordIdentityProvider' and openshift.master.manage_htpasswd | bool
+  with_items: "{{ openshift.master.identity_providers }}"
+
+- name: Ensure htpasswd file exists
+  copy:
+    dest: "{{ item.filename }}"
+    state: touch
+    mode: 0600
```

Comment:
Does this task show as changed every time? From the docs it appears that would be the case. I'd prefer that we didn't update the file time stamps or report changed on every run.